src/server/pps/server/api_server.go

```diff
@@ -2348,7 +2272,8 @@ func (a *apiServer) inspectPipelineInTransaction(txnCtx *txncontext.TransactionC
 		}
 		return nil, err
 	}
-	pipelinePtr.SpecCommit.ID = ancestry.Add(pipelinePtr.SpecCommit.ID, ancestors)
+	// TODO(global ids): how should we treat ancestry here?  Alias commits are going to gum it up.
```

Comment:
I guess this is a case where having the alias commits not be in the commit chain would be helpful.